datasetinsights/stats/visualization/plots.py

```diff
@@ -370,3 +372,101 @@ def rotation_plot(df, x, y, z=None, max_samples=None, title=None, **kwargs):
         )
     )
     return fig
+
+
+def model_performance_comparison_box_plot(
+    title,
```

Comment:
Avoid positional argument when you have two many arguments.
Ex:  mean_ap_base=None